index.html

```diff
@@ -3,8 +3,170 @@
   <head>
     <meta charset="utf-8">
     <title>becharit-2018-team6</title>
+    <link href="https://use.fontawesome.com/releases/v5.0.6/css/all.css" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/header.css">
+    <link rel="stylesheet" href="owl-carousel/owl.carousel.min.css">
   </head>
   <body>
+    <!-- Header -->
+    <div class="header">
+        <header id="header" class="navbar header--fixed hide-from-print headroom headroom--not-bottom headroom--pinned headroom--top">
+            <header class="navbar-scroll">
+                <header class="nav-top">
+                  <div class="continer">
+                      <div class="phone DIB">
+                          <p>
+                            <i class="fas fa-phone-volume"></i> +96-355-005-478-20</p>
+                      </div>
 
+                      <div class="email DIB">
+                          <p><i class="fas fa-envelope"></i> team-6@exampel.com</p>
+                      </div>
+
+                      <input class="btn1" type="button" name="DONET" value="DONET">
+
+                      <div class="search DIB">
+                          <input type="search" name="search" placeholder="search">
+                          <button type="button" name="button">search</button>
+                      </div>
+
+                      <div class="icons DIB ">
```

Comment:
change color + float right 
use this website for color https://www.materialui.co/socialcolors

Reply:
and add div for language-btn+currency-btn+join-top